record-and-playback/core/lib/recordandplayback.rb

```diff
@@ -240,8 +240,17 @@ def self.rap_scripts_path
   def self.read_props
     return @props if @props
 
+    filepathRecOverride = "/etc/bigbluebutton/recording.yml"
+    isThereOverride = File.file?(filepathRecOverride)
```

Comment:
better call it `hasOverride`?

Reply:
It is better indeed, thank you for the feedback.